src/css/variables.less

```diff
@@ -16,6 +16,9 @@
 @link-color: #ffffff;
 @dropdown-bg-color: #ffffff;
 @dropdown-separator-color: #DADCDE;
+@table-header-color:#46494D;
+@table-highlight-color:#3A3C40;
+@table-border-color:#3C3F43;
```

Comment:
Cool! I wonder if we could get `table-highlight-color` by using a `ligthen()` function on the darker bg color...